conf/modules/eff_scheduling_nederdrone.xml

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+<module name="eff_scheduling_nederdrone" dir="ctrl">
+  <doc>
+    <description>Interpolation of control effectivenss matrix
+of the Nederdrone.
+
+If instead using online adaptation is an option, be sure to
+not use this module at the same time!</description>
+  </doc>
+  <settings>
+    <dl_settings>
+      <dl_settings name="eff_scheduling">
+        <dl_setting shortname="tip_prop_always_on" var="sched_tip_props_always_on" min="0" step="1" max="1"/>
+        <dl_setting shortname="tip_prop_lower_pitch" var="sched_tip_prop_lower_pitch_limit_deg" min="-90" step="1" max="0"/>
+        <dl_setting shortname="tip_prop_upper_pitch" var="sched_tip_prop_upper_pitch_limit_deg" min="-90" step="1" max="0"/>
+        <dl_setting shortname="sched_ratio_tip_props" var="sched_ratio_tip_props" min="0" step="0.1" max="1"/>
+        <dl_setting shortname="thrust_eff_scaling" var="thrust_eff_scaling" min="0.6" step="0.01" max="1.4"/>
+        <dl_setting shortname="all_act_fwd_sched" var="all_act_fwd_sched" min="0" step="1" max="1" values="OFF|ON"/>
+      </dl_settings>
+    </dl_settings>
+  </settings>
+  <header>
+    <file name="eff_scheduling_nederdrone.h"/>
+  </header>
+  <init fun="ctrl_eff_scheduling_init()"/>
+  <periodic fun="ctrl_eff_scheduling_periodic()" freq="20.0"/>
+  <makefile>
+    <file name="eff_scheduling_nederdrone.c"/>
+    <test/>
```

Comment:
test is failing because a firmware should be specified (e.g. `<test firmware="rotorcraft">`) and you also need to define some values for `INDI_OUTPUTS` and `INDI_NUM_ACT`, just to pass compilation.

Reply:
I added the firmware, the other defines should always be there as it should always be compiled with stabilization_indi. On my computer all Nederdrone ariframes including the CubeOrange compile, but we can wait for semaphore to see if it is ok now

Reply:
Sorry, I wasn't clear maybe, it is the `test` that requires the firmware option (you can also keep the one in makefile section it is okay). So for you it would look like:
```
<test firmware="rotorcraft">
  <define name="INDI_OUTPUTS" value="4"/>
  <define name="INDI_NUM_ACT" value="4"/>
</test>
```
You can of course change `4` by any number.
If you want to test the modules, you can run  `make test_modules` (it will test all modules) or for a single one:
```
./tests/modules/test_modules.py -f <global_path_to_module>.xml
```

Reply:
Thanks, sorry indeed I was not aware of the compilation test for the modules!
Actually there are quite a few more defines that the module depends on, like control effectiveness values and radio channels. At that point I'm not sure if making the test is still very useful, as the module is there specifically for the Nederdrone airframes which have it included, so it will be compiled in the airframe tests anyway. Is it ok to omit the test?

Reply:
you're the one using it, it's up to you ;)